src/supporting-data/landing-page/LandingPage.tsx

```diff
@@ -0,0 +1,280 @@
+import cn from 'classnames';
+import { Button } from 'franklin-sites';
+import { ChangeEvent, useId } from 'react';
+import { Link } from 'react-router-dom';
+
+import { Location, LocationToPath } from '../../app/config/urls';
+import { useFormLogic } from '../../contact/adapters/contactFormAdapter';
+import SupportingDataIllustration from '../../images/Supporting_data_illustration.jpg';
+import HTMLHead from '../../shared/components/HTMLHead';
+import { SingleColumnLayout } from '../../shared/components/layouts/SingleColumnLayout';
+import styles from './styles/landing-page.module.scss';
+
+const metaDescription =
+  'The Supporting Data section enhances UniProt by providing curated and automated datasets that enrich protein entries with detailed annotations, including keywords, localization, functions, taxonomy, and literature citations.';
```

Comment:
```suggestion
  'Supporting Data enhances UniProt by providing curated and automated datasets that enrich protein entries with detailed annotations, including keywords, localization, functions, taxonomy, and literature citations.';
```
I wasn't sure what the section was exactly as it consists of different namespaces.